Attachment_Downloader/credentials.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"1004827760520-vhe76mj8bnhl2djbi911hul9ebiuar5p.apps.googleusercontent.com","project_id":"attachment-downl-1615836904083","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"jiOeNPFSLKb6TeAGge-Fta-5","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
What is this file?

Reply:
So to be able to import ezgmail , you have to first download credential.json from Google developer website. This step is mentioned in setup instructions section of readme file.

Reply:
Don't you think this file was not supposed to be uploaded here as anyone can misuse this and make changes to your account?

Reply:
Yup will delete this file

Reply:
Also, to be on safer side, revoke this token in your account settings in Gmail as one can still access these commits

Reply:
Okay, will do

Reply:
I've made the necessary changes. Please check.